src/core/qgsattributeeditorelement.h

```diff
@@ -134,10 +134,23 @@ class CORE_EXPORT QgsAttributeEditorElement SIP_ABSTRACT
      */
     void setShowLabel( bool showLabel );
 
+    /**
+     * Returns the (possibly empty) label expression
```

Comment:
Can you clarify the implications of "empty" (i.e. does that mean replace with static text?)

Reply:
no, that is just the getter for the expression member variable. I just wanted to point out that it could be an empty string if there is no expression set.
Do you think I should elaborate more in the docs?

Reply:
Yes, a bit more text would be nice.
The question would probably be more interesting for the setter. If empty string means "expression disabled, static alias will be used" or "No labeling takes place".